android/src/com/mapswithme/maps/search/FilterFragment.java

```diff
@@ -46,9 +47,8 @@
   @SuppressWarnings("NotNullFieldNotInitialized")
   @NonNull
   private PriceFilterView mPrice;
-  @NonNull
-  private final Drawable mTagsDecorator
-      = ContextCompat.getDrawable(MwmApplication.get(), R.drawable.divider_transparent_half);
+  @DrawableRes
+  private final int mTagsDecorator = R.drawable.divider_transparent_half;
```

Comment:
"Can be converted to a local variable"(compiler)

Reply:
done